sonar-text-dotnet/pipeline_scripts/build_and_pack_solution.ps1

```diff
@@ -22,6 +22,20 @@ jf dotnet-config --global --server-id-resolve repox --repo-resolve $nugetFeed
 $signAssembly = "$env:CIRRUS_BRANCH" -eq "master" -or "$env:CIRRUS_BRANCH".startsWith("branch-")
 Write-Host "Should the assembly be signed: $signAssembly"
 
+Write-Host "Locating SignTool.exe"
+$searchRoot = ${env:ProgramFiles(x86)} + '\Windows Kits\10\bin\10*'
+$exeName = 'signtool.exe'
+$signtool = Get-ChildItem -Path $searchRoot -Filter $exeName -Recurse -ErrorAction SilentlyContinue -Force | Select -Last 1
+if (!$signtool)
+{
+throw 'Unable to find ' + $exeName + ' under ' + $searchRoot
+}
+
+Write-Host 'Resolving paths '
+$env:PFX_PATH = resolve-path ${env:PFX_PATH}
```

Comment:
I have mixed feelings about overriding the environment variables.
I would go for:
```suggestion
$pfxPath = resolve-path $env:PFX_PATH
```